.github/workflows/lsst-tests.yml

```diff
@@ -60,3 +59,19 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           files: ./coverage.xml
+          fail_ci_if_error: false
+
+  notify-on-failure:
+    runs-on: ubuntu-latest
+    if: ${{ always() && (github.event_name == 'push' || github.event_name == 'schedule') && needs.lsst-tests.result == 'failure' }}
+    needs:
+      - lsst-tests
+    steps:
+    - uses: actions/checkout@v2
+    - name: Slack Notification
```

Comment:
I've never used this before so how does it know which channel to write to?

Reply:
I can add the app to any channel, or from the App management page I can add it to any channel.